invoices/invoiceregistry.go

```diff
@@ -41,6 +49,43 @@ type HodlEvent struct {
 	AcceptHeight int32
```

Comment:
It's a bit difficult to see what this type is suppsed to do without looking at the comment. May I suggest to rename it to something more descriptive? eg. `htlcAutoReleaseEvent`. And maybe also for members. What does `hash` and `key` point to?

Reply:
comments added